<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="login">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>280</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ID">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>150</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="pass">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>210</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoLoginPage">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>121</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gotoSignupPage">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>300</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SIGNUP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="firstname">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LastName">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>170</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="SetId">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>220</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="SetPass">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>263</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfirmPass">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>310</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="signup">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>350</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SIGNUP</string>
    </property>
   </widget>
   <widget class="QLabel" name="loginAndSignupLabel">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>-25</y>
      <width>811</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="LoginPage">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>-25</y>
      <width>801</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="SignUpPage">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>-25</y>
      <width>801</width>
      <height>471</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../8.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="back">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <zorder>loginAndSignupLabel</zorder>
   <zorder>LoginPage</zorder>
   <zorder>SignUpPage</zorder>
   <zorder>firstname</zorder>
   <zorder>LastName</zorder>
   <zorder>SetId</zorder>
   <zorder>SetPass</zorder>
   <zorder>ConfirmPass</zorder>
   <zorder>signup</zorder>
   <zorder>gotoLoginPage</zorder>
   <zorder>gotoSignupPage</zorder>
   <zorder>login</zorder>
   <zorder>pass</zorder>
   <zorder>ID</zorder>
   <zorder>back</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
